### ISSUE TITLE
CheckNPCAlly Error

### ISSUE THREAD

**Issue by Hazeofdream** [Balimbanana/SM-Synergy#13]
![image](https://github.com/Balimbanana/SM-Synergy/assets/74027566/9e2cd41c-bbce-4cbd-80ad-b8ccc329f340)

Causes a full error of the plugin https://github.com/Balimbanana/SM-Synergy/blob/master/plugins/synsweps.smx

**Comment by Balimbanana** [comment]
Yes, that native is part of HealthDisplay, instead of having to write out the friend/foe detection twice, that part is in HealthDisplay.
